lib/config/octane.js

```diff
@@ -1,46 +1,16 @@
 'use strict';
 
 module.exports = {
+  extends: ['recommended'],
   rules: {
-    'deprecated-render-helper': true,
-    'link-rel-noopener': 'strict',
-    'link-href-attributes': true,
-    'no-abstract-roles': true,
     'no-action': true,
     'no-args-paths': true,
-    'no-attrs-in-components': true,
     'no-curly-component-invocation': {
       noImplicitThis: true,
       requireDash: false,
     },
-    'no-debugger': true,
-    'no-duplicate-attributes': true,
-    'no-extra-mut-helper-argument': true,
-    'no-html-comments': true,
-    'no-implicit-this': true,
```

Comment:
@rwjblue shouldn't this remain here? 🤔

Reply:
Crappola, yeah. I'll fix. I removed it because I was comparing `next` at the time, _then_ I merged the other PR :sob: